src/pages/markets/instruments/_dex-indices.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Symbol from '../components/helper/_symbol'
+import { dex_indices } from '../static/content/_market-symbols'
+
+const DexIndices = () => {
+    return (
+        <>
+            {dex_indices.map((symbol, index) => (
+                <Symbol key={index} src={symbol.src} text={symbol.text} />
```

Comment:
Can we use something else instead of index for key. 'index' is kind of the last thing we should use for key.